doc/conf.py

```diff
@@ -16,13 +16,14 @@
 
 
 # -- Project information -----------------------------------------------------
+from ..cve_bin_tool.version import VERSION
```

Comment:
You can't to a relative import here. You need to uncomment line 15 and change `.` to `..`. Then you can do the following:

```suggestion
from cve_bin_tool.version import VERSION
```

This is because relative imports only work so long as there is an `__init__.py` file, which there isn't in the top level directory of the source tree (and there never should be).

Reply:
@pdxjohnny  if we use the same virtual environment as cve-binary tool for fetching this VERSION value then its okay to use
``from cve_bin_tool.version import VERSION ``
Otherwise, it will give import module errors for all the packages used in cve_bin_tool/version.py
We can scrap the values similar to setup.py though